pkg/api/openshiftcluster.go

```diff
@@ -258,12 +258,13 @@ type OIDCIssuer string
 type ClusterProfile struct {
 	MissingFields
 
-	PullSecret           SecureString         `json:"pullSecret,omitempty"`
-	Domain               string               `json:"domain,omitempty"`
-	Version              string               `json:"version,omitempty"`
-	ResourceGroupID      string               `json:"resourceGroupId,omitempty"`
-	FipsValidatedModules FipsValidatedModules `json:"fipsValidatedModules,omitempty"`
-	OIDCIssuer           OIDCIssuer           `json:"oidcIssuer,omitempty"`
+	PullSecret                    SecureString         `json:"pullSecret,omitempty"`
+	Domain                        string               `json:"domain,omitempty"`
+	Version                       string               `json:"version,omitempty"`
+	ResourceGroupID               string               `json:"resourceGroupId,omitempty"`
+	FipsValidatedModules          FipsValidatedModules `json:"fipsValidatedModules,omitempty"`
+	OIDCIssuer                    OIDCIssuer           `json:"oidcIssuer,omitempty"`
+	BoundServiceAccountSigningKey SecureString         `json:"boundServiceAccountSigningKey,omitempty"`
```

Comment:
we're not going to populate this on older clusters, correct?  So this field should only be set on WI clusters?

Reply:
Yes this value will only be generated and populated for MIWI clusters as part of OIDC blob creations for the clusters.

Reply:
Do you want to make it a pointer then?

Reply:
Changed it SecureBytes now.

Reply:
Changed it to *SecureString now, please review the [commit](https://github.com/Azure/ARO-RP/pull/3564/commits/6067a9dc670b123a2845e812d0dfa3c23c76acc7)

Reply:
Resolving this thread since the change has been made and the motivation for the change is clear.